editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-element-renderer-utils.tsx

```diff
@@ -751,10 +762,12 @@ export function utopiaCanvasJSXLookup(
 }
 
 function runJSExpression(
-  filePath: string,
-  requireResult: MapLike<any>,
   block: JSExpression,
+  elementPath: ElementPath | null,
   currentScope: MapLike<any>,
+  renderContext: RenderContext,
+  uid: string | undefined,
+  codeError: Error | null,
```

Comment:
I would leave a comment here that this should probably deleted in a further refactor